BaragonService/src/main/java/com/hubspot/baragon/service/config/BaragonConfiguration.java

```diff
@@ -46,6 +46,10 @@
   @Min(1)
   private int agentMaxAttempts = 5;
 
+  @JsonProperty("agentMaxRequestTime")
+  @Min(10000)
+  private long agentMaxRequestTime = 60000;
```

Comment:
rest of the config items have the units at the end, so we should probably name this `agentMaxRequestTimeMs`. i'd also argue that the name is a little confusing -- what do you think about calling it `agentRequestTimeoutMs` instead?